db/schema.rb

```diff
@@ -36,14 +36,7 @@
     t.string "confirmation_token"
     t.datetime "confirmed_at"
     t.datetime "confirmation_sent_at"
-    t.string "invitation_token"
-    t.datetime "invitation_created_at"
-    t.datetime "invitation_sent_at"
-    t.datetime "invitation_accepted_at"
-    t.integer "invitation_limit"
-    t.string "invited_by_type"
-    t.bigint "invited_by_id"
-    t.integer "invitations_count", default: 0
+    t.string "unconfirmed_email"
```

Comment:
Why are all the devise fields missing? Seems like something else happened. Was devise removed?

Reply:
Those fields snuck into the schema.rb at some point when Invitable was going to be added but it was removed to be implemented later. The invitation fields stuck around in schema since they got committed in there but without any migrations. We'll need to fix this in a PR.

Reply:
@BrendaPenn and @colinloretz you *could* checkout master, checkout a branch to update the schema, run `db:migrate` to make these changes, commit, PR, then rebase this branch. 

I can show you how to do that if you need it.

Reply:
Thanks for the review @elskwid ! I'll clean up this PR :)